bytecomp/semantics_of_primitives.ml

```diff
@@ -142,14 +142,17 @@ let for_primitive (prim : Lambda.primitive) =
   | Pint_as_pointer -> No_effects, No_coeffects
   | Popaque -> Arbitrary_effects, Has_coeffects
   | Ploc _ ->
-    Misc.fatal_error "[Ploc] should have been eliminated by [Translcore]"
+      (* Removed by [Translcore]. *)
+      No_effects, No_coeffects
   | Prevapply
-  | Pdirapply
+  | Pdirapply ->
```

Comment:
The indentation is coherent with was ocp-indent does, considering the definitions in the toplevel .ocp-indent.  It is also coherent with the style used elsewhere in the compiler (except in newest flambda code, I guess), and I find it more readable than the style used elsewhere in this file.  Since the file moves to bytecomp, I'd rather apply a style similar to other files in this directory, i.e. apply ocp-indent on the whole file.  Would you oppose to that?